subsys/fs/ext2/Kconfig

```diff
@@ -18,7 +18,7 @@ if FILE_SYSTEM_EXT2
 menu "Ext2 file system Settings"
 	visible if FILE_SYSTEM_EXT2
 
-config MAX_FILES
+config EXT2_MAX_FILES
```

Comment:
This is change to Kconfig that is available to an user, there should be release note on change in Kconfig name and migration guide provided.

Reply:
This hasn't been addressed properly -- @de-nordic is right, and the important part is the migration guide actually. I don't think there is anything worth mentioning in release notes so please just move the bullet point you've added to release notes to migration guide instead. Thanks!

Reply:
@kartben  OK. So for API changes, etc we no longer do the release notes, rather migration guide?